server/views/match/search.njk

```diff
@@ -1,15 +1,137 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
 {% extends "../partials/layout.njk" %}
 
 {% set pageTitle = applicationName + " - " + pageHeading %}
 
 {% block content %}
 
-	<h1 class="govuk-heading-xl">Find a bed</h1>
+	<h1 class="govuk-heading-m">{{person.name}}</h1>
+	<dl class="search-identity-table govuk-body">
+
+		<div class="search-identity-table__column">
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Tier</strong>
+				</dt>
+				<dd>A1</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>CRN
+				</strong>
+				</dt>
+				<dd>{{person.crn}}</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>DOB
+				</strong>
+				</dt>
+				<dd>{{person.dateOfBirth}}</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Expected arrival date</strong>
+				</dt>
+				<dd>{{formatDate(startDate)}}</dd>
+			</div>
+		</div>
+		<div class="search-identity-table__column search-identity-table__table-cell__last-col">
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Assessment</strong>
+
+				</dt>
+				<dd>
+					<a href="{{assessmentPath}}">View <span class="govuk-visually-hidden">assessment</span></a>
+				</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Application</strong>
+				</dt>
+				<dd>
+					<a href="{{applicationPath}}">View <span class="govuk-visually-hidden">Application</span></a>
```

Comment:
Probably best to have lower case `Application` here, as it's lowercase for `assessment`